src/Files.App/Commands/Manager/HotKeyChangedEventHandler.cs

```diff
@@ -0,0 +1,4 @@
+﻿namespace Files.App.Commands
+{
+	public delegate void HotKeyChangedEventHandler(ICommandManager manager, HotKeyChangedEventArgs e);
```

Comment:
This feels over engineered, does it really need it's own class?

Reply:
It is not a class, only a delegate.

Reply:
True, but the main point is that it doesn't need a separate file.

Reply:
The delegate is removed.